k8s/namespaces/jenkins/patches/ptl/cluster-00/jenkins.yaml

```diff
@@ -133,10 +133,10 @@ spec:
                     title: Hearing Management Interface
                 - buildMonitor:
                     includeRegex: >-
-                      ^HMCTS_.*PRE-.*\/.+\/master
+                      ^HMCTS_.*PRE-.*\/pre\-.+)\/master
```

Comment:
again why?
this is currently picking up everything in the HMCTS_PRE folder on jenkins

Unless you need to filter a legacy project out (which you should probably do at the org config level).
Or you want to split something like infra and app out (unlikely)

I don't think you need this